Samples/Server/Unity-FractalStream/Assets/Scripts/WebRTCServer.cs

```diff
@@ -387,17 +462,35 @@ private void SetupActiveEyes()
             if (this.IsStereo)
             {
                 this.LeftEye.stereoTargetEye = StereoTargetEyeMask.Left;
-                this.RightEye.enabled = true;
+                this.LeftEye.enabled = false;
+                this.RightEye.enabled = false;
             }
             else
             {
                 // none means use the eye like a single camera, which is what we want here
                 this.LeftEye.stereoTargetEye = StereoTargetEyeMask.None;
+                this.LeftEye.enabled = false;
                 this.RightEye.enabled = false;
             }
 
             // update our last setup visible eye value
             this.lastSetupVisibleEyes = this.IsStereo;
+
+            // notify the plugin of the potential stereo change as well
+            if (this.Plugin != null)
+            {
+                this.Plugin.EncodingStereo = this.IsStereo;
+            }
+        }
+
+        /// <summary>
+        /// Sends frame buffer.
+        /// </summary>
+        private IEnumerator SendFrame(bool isStereo)

```

Comment:
this should be encapsulated in the `Plugin` class

Reply:
Agree. To be updated.

Reply:
can we now remove this `SendFrame` instance and instead call `StartCoroutine(Plugin.SendFrame)` above? that was the goal with my request here